app/admin/textrepl/controller.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+import EmberValidations from 'ember-validations';
+
+export default Ember.Controller.extend(EmberValidations, {
+  hideCancelButton: true,
+  updateCapability: 'update_config',
+
+  createExpansion: function() {
```

Comment:
we should be using `on(function(){})` format as the current syntax will eventually be removed. you can export the on as well by using `const { on } = Ember`